Source/VisualStudioTools/Private/VSTestAdapterCommandlet.cpp

```diff
@@ -2,134 +2,153 @@
 
 #include "VSTestAdapterCommandlet.h"
 
-#include <Runtime/Core/Public/Async/TaskGraphInterfaces.h>
-#include <Runtime/Core/Public/Containers/Ticker.h>
-#include <fstream>
```

Comment:
`#include <fstream>` was removed by the classes are still used, so I'm assuming this is working because some other file is bringing the indirect dependency. Consider building the plugin locally with the Unity Build disabled [1] to ensure it's going to work for everyone. We got some issues with that recently.

[1] Add `bUseUnityBuild = false` to the .Build.cs file for our module. Just remember to revert it later if pushing more changes.

Reply:
Works